extensions/blocks/podcast-episodes/edit.js

```diff
@@ -3,49 +3,139 @@
+import { BlockControls, BlockIcon, InspectorControls } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
+import { isURL } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
-import icon from './icon';
-
-function PodcastEpisodesEdit( {
-	attributes,
-	className,
-	noticeOperations,
-	noticeUI,
-	setAttributes,
-} ) {
-	/**
-	 * Write the block editor UI.
-	 *
-	 * @returns {object} The UI displayed when user edits this block.
-	 */
-	const [ notice, setNotice ] = useState();
-
-	/* Call this function when you want to show an error in the placeholder. */
-	const setErrorNotice = () => {
-		noticeOperations.removeAllNotices();
-		noticeOperations.createErrorNotice( __( 'Put error message here.', 'jetpack' ) );
+import { edit, queueMusic } from './icons/';
+
+const DEFAULT_MIN_ITEMS = 1;
+const DEFAULT_MAX_ITEMS = 10;
+
+class PodcastEpisodesEdit extends Component {
+	constructor() {
+		super( ...arguments );
+		this.toggleAttribute = this.toggleAttribute.bind( this );
+		this.onSubmitURL = this.onSubmitURL.bind( this );
+
+		this.state = {
+			editing: ! this.props.attributes.url,
+			urlError: '',
+		};
+	}
+
+	toggleAttribute( propName ) {
+		return () => {
+			const value = this.props.attributes[ propName ];
+			const { setAttributes } = this.props;
+
+			setAttributes( { [ propName ]: ! value } );
+		};
+	}
+
+	onSubmitURL( event ) {
+		event.preventDefault();
+
+		if ( this.props.attributes.url ) {
+			const isValidURL = isURL( this.props.attributes.url );
+			this.setState( {
+				editing: ! isValidURL,
+				urlError: ! isValidURL
+					? __( 'The URL you entered is invalid. Please check and try again.', 'jetpack' )
+					: '',
+			} );
+		}
+	}
+
+	handleSSRError = () => {
+		return <p>{ __( 'Failed to load Block', 'jetpack' ) }</p>;
 	};
 
-	return (
-		<div className={ className }>
-			<Placeholder
-				label={ __( 'Podcast Episodes', 'jetpack' ) }
-				instructions={ __( 'Instructions go here.', 'jetpack' ) }
-				icon={ <BlockIcon icon={ icon } /> }
-				notices={ noticeUI }
-			>
-				{ __( 'User input goes here?', 'jetpack' ) }
-			</Placeholder>
-		</div>
-	);
+	render() {
+		const { url, itemsToShow } = this.props.attributes;
+		const { attributes, setAttributes } = this.props;
+
+		if ( this.state.editing ) {
+			return (
+				<Placeholder
+					icon={ <BlockIcon icon={ queueMusic } /> }
+					label={ __( 'Podcast Episodes', 'jetpack' ) }
+					instructions={ __( 'Paste a link to your Podcast RSS feed.', 'jetpack' ) }
+				>
+					<form onSubmit={ this.onSubmitURL }>
+						{ this.state.urlError && <Notice>{ this.state.urlError }</Notice> }
+						<TextControl
+							placeholder={ __( 'Enter URL here…', 'jetpack' ) }
+							value={ url || '' }
+							onChange={ value => setAttributes( { url: value } ) }
+							className={ 'components-placeholder__input' }
+						/>
+						<Button isSecondary type="submit">
```

Comment:
```suggestion
						<Button isPrimary type="submit">
```

To make it the same as the core RSS block.

<img width="570" alt="Screenshot 2020-03-12 at 13 57 17" src="https://user-images.githubusercontent.com/156676/76523864-68c05400-6469-11ea-81be-c93383dfe407.png">

Reply:
This looks to have been done.

Reply:
I can still see "secondary" in the diff

Reply:
but you can no longer use the Github suggestion as it points to the old file location

Reply:
Looks fixed to me here

https://github.com/Automattic/jetpack/pull/14948/files#diff-ac95db14f00bcb6ae1960269ae9121cfR81

Reply:
I guess udated here https://github.com/Automattic/jetpack/pull/14948/commits/5d63e94d80feeb9021ac3347fe2417e0bb15672c